CI/travis.osx.after_success.sh

```diff
@@ -2,7 +2,8 @@
 
 set -e
 
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+if [ "${TRAVIS_EVENT_TYPE}" == "cron" ] \
+  && [ "${CC}" != "clang" ] && [ "${Q_OR_C_MAKE}" != "qmake" ]; then
```

Comment:
We have only clang on macOS, so the condition is unnecessary.

Reply:
Fixed.